nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -280,6 +280,13 @@ public class HandleHttpRequest extends AbstractProcessor {
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .expressionLanguageSupported(ExpressionLanguageScope.NONE)
             .build();
+    public static final PropertyDescriptor REQUEST_HEADERS_MAX_SIZE = new PropertyDescriptor.Builder()
+            .name("HTTP Headers Maximum Size")
```

Comment:
Recommend dropping `HTTP` and adding `Request` since this is specific to request headers and not response headers.
```suggestion
            .name("Request Header Maximum Size")
```

Reply:
Agreed!